app/assets/config/decidim_challenges_manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link decidim/challenges/add_challenge.js
+//= link decidim/challenges/admin/challenges_form.js
+//= link_tree ../images/decidim
```

Comment:
are we requiring Javsacripts from the images/ dir?
```suggestion
```

Reply:
Sure not, I'll remove!